src/fiatconnect/FiatDetailsScreen.tsx

```diff
@@ -106,20 +111,52 @@ const FiatDetailsScreen = ({ route, navigation }: Props) => {
 
   useLayoutEffect(() => {
     navigation.setOptions({
-      headerTitle: i18n.t('fiatDetailsScreen.header'),
-      headerLeft: () => <BackButton onPress={onPressBack} />,
+      headerTitle: () => (
+        <View style={headerStyles.header}>
+          <Text style={headerStyles.headerTitle} numberOfLines={1}>
+            {t('fiatDetailsScreen.header')}
+          </Text>
+          <View style={styles.headerSubTitleContainer}>
+            <View style={styles.headerImageContainer}>
+              <Image
+                testID="headerProviderIcon"
+                style={styles.headerImage}
+                source={{ uri: quote.getProviderIcon() }}
+                resizeMode="contain"
+              />
+            </View>
+            <Text numberOfLines={1} style={headerStyles.headerSubTitle}>
```

Comment:
Did you encounter problems with the text exceeding the available space in the header title?

Reply:
The number of lines option makes it so that the field doesn't wrap around and changes to `...` if the text exceeds the available width. The headerTitle does have a maxWidth of total width * 0.6, we could copy the same to subtitle. Below are screenshots without and with maxWidth set to 0.6 * total width on the subtitle.

Without maxWidth on subtitleContainer:
<img src="https://user-images.githubusercontent.com/5062591/183494770-79c763cd-afab-4385-b3dd-2c9abb5757d6.png" alt="withoutMaxWidth" width="400"/>


With maxWidth on subtitleContainer:
<img src="https://user-images.githubusercontent.com/5062591/183494769-1ce44b21-b47b-4d4b-9919-f2dea20f8f2b.png" alt="withMaxWidth" width="400"/>

I don't feel too strongly about adding this. We should just keep provider names short so they don't overflow. What do you think?